aleth-key/KeyAux.h

```diff
@@ -692,7 +692,7 @@ class KeyCLI
 			<< "    --force-nonce <n>  Override the nonce for any transactions to be signed." << endl
 			<< endl
 			<< "Encryption configuration:" << endl
-			<< "    --kdf <kdfname>  Specify KDF to use when encrypting (default: sc	rypt)" << endl
```

Comment:
<< "    --kdf <kdfname>  Specify KDF to use when encrypting (default: scrypt)" << endl